docs/sources/flow/reference/components/loki.process.md

```diff
@@ -0,0 +1,192 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/loki.process
+title: loki.process
+---
+
+# loki.process
+
+`loki.process` receives log entries from other loki components, applies one or
+more processing _stages_, and forwards the results to the list of receivers
+in the component's arguments.
+
+A stage is a multi-purpose tool that can parse, transform, and filter log
+entries before they're passed to a downstream component. These stages are
+applied to each log entry in order of their appearance in the configuration
+file. All stages within a `loki.process` block have access to the log entry's
+label set, the log line, the log timestamp, as well as a shared map of
+'extracted' values; this allows results of one stage to be used in a subsequent
+one.
+
+Multiple `loki.process` components can be specified by giving them
+different labels.
+
+## Usage
+
+```river
+loki.process "LABEL" {
+  forward_to = RECEIVER_LIST
+
+  stage {
+    ...
+  }
+  ...
+}
+```
+
+## Arguments
+
+`loki.process` supports the following arguments:
+
+Name              | Type                 | Description                                      | Default | Required
+----------------- | -------------------- | ------------------------------------------------ | ------- | --------
+`forward_to`      | `list(LogsReceiver)` | Where to forward log entries after processing. | | yes
+
+## Blocks
+
+The following blocks are supported inside the definition of `loki.process`:
+
+Hierarchy      | Block      | Description | Required
+-------------- | ---------- | ----------- | --------
+stage          | [stage][]  | Processing stage to run. | no
+stage > json   | [json][]   | Configures a JSON processing stage.  | no
+stage > labels | [labels][] | Configures a Labels processing stage. | no
```

Comment:
I think "Labels" should be lowercase here.

Reply:
Ok, keeping this in mind as a convention for the stages that we'll gradually add as well.